content/en/platform/corda/1.3/cenm/deployment-kubernetes.md

```diff
@@ -293,21 +293,32 @@ You will find the truststore password in the `signer/files/pki.conf`, where the
 
 ### Display logs
 
-Each CENM service has a dedicated sidecar to display live logs from the `log/` directory.
+Each CENM service has dedicated sidecar to displays live logs from `log/` folder.
```

Comment:
"...to displays live logs from `log/` folder."

Is there a typo here? Should be 'to display` no?

also why folder not directory?